packages/mantine/jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
You will need to convert this to a `.json`  file due to the `package.json` changes I'm requesting